Assets/Countly/Plugins/CountlySDK/Countly.cs

```diff
@@ -365,7 +365,7 @@ private void CreateListOfIBaseService()
             _listeners.Add(Location);
             _listeners.Add(Consents);
             _listeners.Add(StarRating);
-            _listeners.Add(UserDetails);
+            _listeners.Add((UserDetailsCountlyService) UserDetails);
```

Comment:
I have inspected a little abstract design. What is our goal for those I don't know but it is good.
But looks like we are moving away from abstraction. If that is the case, probably listener structure will be refactored also.

If not I did not like the specific casting. But it will be fixed soon I guess